conf4j-core/src/main/java/com/sabre/oss/conf4j/converter/DefaultTypeConverters.java

```diff
@@ -25,15 +25,42 @@
 package com.sabre.oss.conf4j.converter;
 
 import java.util.List;
+import java.util.function.Function;
 
 import static java.util.Arrays.asList;
+import static java.util.Collections.singletonList;
+import static java.util.Collections.unmodifiableList;
+import static java.util.Objects.requireNonNull;
 
 /**
  * A utility class which configures {@link TypeConverter} with the default set of type converters.
  */
 public final class DefaultTypeConverters {
+    private static final List<TypeConverter<?>> DEFAULT_BASE_CONVERTERS = unmodifiableList(asList(
+            new StringConverter(),
```

Comment:
It looks the list of converters is not complete - there were a few changes in this are recently (at least `CurrencyConverter` is missing).